### ISSUE TITLE
请问为什么我BL+IADM for RGBT Crowd Counting训练准确率达不到这么高。

### ISSUE THREAD

**Issue by lyccol** [chen-judge/RGBTCrowdCounting#2]
谢谢大佬提供的代码，我在复现的时候遇到了些问题。

我使用的单张3090，其他全部使用默认参数进行直接训练。训练过程为：

...................
08-27 08:31:48 -----Epoch 197/199-----
08-27 08:33:28 Epoch 197 Train, Loss: 6.54, GAME0: 3.61 MSE: 7.36, Cost 99.3 sec
08-27 08:33:43 Epoch 197 Val200, GAME0 10.53 GAME1 15.08 GAME2 19.54 GAME3 27.40 MSE 17.58 Re 0.1624, 
08-27 08:33:43 *** Best Val GAME0 10.531 GAME3 27.315 model epoch 197
08-27 08:34:38 Epoch 197 Test800, GAME0 17.60 GAME1 21.88 GAME2 26.41 GAME3 34.43 MSE 30.26 Re 0.2425, 
08-27 08:34:38 -----Epoch 198/199-----
08-27 08:36:18 Epoch 198 Train, Loss: 6.21, GAME0: 3.40 MSE: 6.97, Cost 99.9 sec
08-27 08:36:34 Epoch 198 Val200, GAME0 12.27 GAME1 16.63 GAME2 20.74 GAME3 29.18 MSE 24.31 Re 0.1924, 
08-27 08:36:34 -----Epoch 199/199-----
08-27 08:38:15 Epoch 199 Train, Loss: 6.49, GAME0: 3.68 MSE: 6.53, Cost 101.2 sec
08-27 08:38:30 Epoch 199 Val200, GAME0 10.66 GAME1 15.33 GAME2 19.65 GAME3 27.63 MSE 15.74 Re 0.1970, 

# ./test.sh 
testing...
/root/anaconda3/lib/python3.7/site-packages/torch/nn/functional.py:3672: UserWarning: nn.functional.upsample_bilinear is deprecated. Use nn.functional.interpolate instead.
  warnings.warn("nn.functional.upsample_bilinear is deprecated. Use nn.functional.interpolate instead.")
Test800, GAME0 17.60 GAME1 21.88 GAME2 26.41 GAME3 34.43 MSE 30.26 Re 0.2425, 

结果只有17.6。请问是否为我实验参数的问题。

**Comment by chen-judge** [comment]
谢谢关注我们的工作。参数初始化、样本增强等有一定随机性，可以再跑几次试试。我们后来也有跑出比paper更好的结果。

**Comment by lyccol** [comment]
感谢感谢，不好意思，还想确认一下，请问一下实验默认的参数是给，模型BL+IADM 和  RGBT-CC这个数据集用的嘛？

例如
gpu数量为1
batch-size : 1
lr :1e-5
weight-deay= 1e-4
等

离投稿不到两周，有点着急复现，实在打扰了。

**Comment by lyccol** [comment]
谢谢  在后面又训练了几十个epoch之后，达到了文中的效果

**Comment by gaiyi7788** [comment]
> 

您好，我想询问一下您有没有试过大一点的batchsize。我一开始用比较大的batchsize训练效果一直很差，看到您这条以后改回1发现loss下降的更好，不知道您有没有过类似的测试，希望可以交流一下，感谢！

**Comment by chen-judge** [comment]
> > 
> 
> 您好，我想询问一下您有没有试过大一点的batchsize。我一开始用比较大的batchsize训练效果一直很差，看到您这条以后改回1发现loss下降的更好，不知道您有没有过类似的测试，希望可以交流一下，感谢！

您好，我没有用其他batchsize做实验，为1的时候确实更容易优化

**Comment by gaiyi7788** [comment]
> > > 
> > 
> > 
> > 您好，我想询问一下您有没有试过大一点的batchsize。我一开始用比较大的batchsize训练效果一直很差，看到您这条以后改回1发现loss下降的更好，不知道您有没有过类似的测试，希望可以交流一下，感谢！
> 
> 您好，我没有用其他batchsize做实验，为1的时候确实更容易优化

好的，非常感谢

**Comment by SDarknessZX** [comment]
> 

想问一下密度图的可视化代码可以发一下吗，我的邮箱893646282@qq.com感激不尽！

**Comment by hqqq666** [comment]
> > 
> 
> 想问一下密度图的可视化代码可以发一下吗，我的邮箱[893646282@qq.com](mailto:893646282@qq.com)感激不尽！

你好，请问能分享一下密度图可视化代码吗？谢谢您！邮箱1050672336@qq.com

**Comment by hqqq666** [comment]
> > 
> 
> 想问一下密度图的可视化代码可以发一下吗，我的邮箱[893646282@qq.com](mailto:893646282@qq.com)感激不尽！
您好，请问能分享一下您的密度图可视化代码吗？非常感谢您！邮箱1050672336@qq.com

**Comment by Joeyqq016** [comment]
> > 
> 
> 想问一下密度图的可视化代码可以发一下吗，我的邮箱[893646282@qq.com]
您好，可以分享一下您改进之后的密度图可视化的代码嘛？我的邮箱joeyhu016@gmail.com。非常感谢！

**Comment by Joeyqq016** [comment]
> > 
> 
> 想问一下密度图的可视化代码可以发一下吗，我的邮箱[893646282@qq.com](mailto:893646282@qq.com)感激不尽！

您好，可以分享一下您改进之后的密度图可视化的代码嘛？我的邮箱[joeyhu016@gmail.com](mailto:joeyhu016@gmail.com)。非常感谢！

**Comment by 7ran** [comment]
> 谢谢 在后面又训练了几十个epoch之后，达到了文中的效果

您好，请问您可以提供一下密度图可视化代码吗，谢谢！

**Comment by 7ran** [comment]
> > 
> 
> 想问一下密度图的可视化代码可以发一下吗，我的邮箱[893646282@qq.com](mailto:893646282@qq.com)感激不尽！

您好，方便分享一下密度图可视化代码吗？谢谢了

**Comment by Kane-Joshua** [comment]
您好，可以分享一下您改进之后的密度图可视化的代码嘛？我的邮箱kanemjo@gmail.com。非常感谢！

**Comment by chen-judge** [comment]
关于密度图可视化，请参考另一个issue [https://github.com/chen-judge/RGBTCrowdCounting/issues/8#issuecomment-1014188822](https://github.com/chen-judge/RGBTCrowdCounting/issues/8#issuecomment-1014188822)
